### PR TITLE
basic content fix

### PR DESCRIPTION
**Pull request by ksathyareddy7**
some of the basic content was fixed including one URL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,24 +155,24 @@ The react app in this quickstart shows us an example of the username/password au
 
 Sometimes, you would want to upload some files to the cloud. This can range from a profile pic for your user or images for things listed on your app. You can securely add, remove, manage, update files such as pictures, videos, documents using Hasura filestore.
 
-You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/users/index.html)
+You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/files/index.html)
```
Comment (Hyperclaw79) [inline README.md:158]: The previous link was correct.

Comment (ksathyareddy7) [reply README.md:158]: The previous link was point to "users and authentication" section which is not relevant to "files" and the new link points to "files" section where the reference is given on how to upload, download, delete, etc... That's the reason I made the above change.

Comment (Hyperclaw79) [reply README.md:158]: No, the link was reference for the usage of `api console`, which is present in the `users/index.html`.
How about including both links?

File: README.md [hunk 2]
```diff
@@ -151,24 +155,24 @@ The react app in this quickstart shows us an example of the username/password au
...
 
 ## Custom Service
 
-There might be cases where you might want to perform some custom business logic on your apis. For example, sending an email/sms to a user on sign up or sending a push notification to the mobile device when some event happens. For this, you would want to create your own custom service which does these for you on the endpoints that you define.
+There might be cases where you might want to perform some custom business logic on your APIs. For example, sending an email/sms to a user on sign up or sending a push notification to the mobile device when some event happens. For this, you would want to create your own custom service which does these for you on the endpoints that you define.
 
-This quickstart comes with one such custom service written in `nodejs` using the `express` framework. Check it out in action at `https://app.cluster-name.hasura-app.io` . Currently, it just returns a "Hello-React" at that endpoint.
+This quickstart comes with one such custom service written in `nodejs` using the `express` framework. Check it out in action at `https://api.cluster-name.hasura-app.io` . Currently, it just returns a "Hello-React" at that endpoint.
```
Comment (coco98) [inline README.md]: We're using `www` as the subdomain in the hello-world quickstart. You can change it to that?

**Comment by coco98** [comment]
@sathya9897 These changes look good. I'm merging them :) Good stuff 👍 
@karthikvt26 You'll have to publish again to hasura.io/hub.

**Pull request merged**